examples/flutter_gallery/lib/demo/material/fab_motion_demo.dart

```diff
@@ -0,0 +1,105 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+const String _explanatoryText =
+  "When the Scaffold's floating action button positioner changes, "
+  'the floating action button animates to its new position';
+
+class FabMotionDemo extends StatefulWidget {
+  static String routeName = '/material/fab-motion';
```

Comment:
this should be const (or final), since i presume it's not supposed to change

Reply:
Done